configuration/rules-dsl.md

```diff
@@ -241,6 +241,10 @@ In openHAB version 3 the System-based Trigger for startlevel had been added, val
 
 Startlevels less than 40 are not available as triggers because the rule engine needs to start up first before it can execute any rules.
 
+A rule triggering for a startlevel and being saved during development, will be reloaded.
+If the expected startlevel (or higher) has already been reached, this rule will trigger again.
+Writing rules with quite low startlevels might be too early, as for instance not all things are online yet.
```

Comment:
This quite generic and more or less stating the obvious. Can you provide an example here to make it more tangible?